src/Illuminate/Queue/DatabaseQueue.php

```diff
@@ -256,7 +256,7 @@ protected function getNextAvailableJob($queue)
     protected function getLockForPopping()
     {
         $databaseEngine = $this->database->getPdo()->getAttribute(PDO::ATTR_DRIVER_NAME);
-        $databaseVersion = $this->database->getConfig('version') ?? $this->database->getPdo()->getAttribute(PDO::ATTR_SERVER_VERSION);
+        $databaseVersion = $this->database->getConfig('options.version') ?? $this->database->getPdo()->getAttribute(PDO::ATTR_SERVER_VERSION);
```

Comment:
Might be better to use this as an alternative should some people rely on the previous key?

```suggestion
        $databaseVersion = $this->database->getConfig('version') ?? $this->database->getConfig('options.version') ?? ?? $this->database->getPdo()->getAttribute(PDO::ATTR_SERVER_VERSION);
```